scripts/pylint_extensions_test.py

```diff
@@ -1143,3 +1143,225 @@ def func3():
 
         with checker_test_object.assertNoMessages():
             temp_file.close()
+
+
+class SingleNewlineAboveArgsCheckerTests(unittest.TestCase):
+
+    def test_checks_newline_above_docstring(self):
```

Comment:
Split up this test and more tests with varying combinations of args/returns/raises with and without spaces. (There should be at least 8 cases to cover all of them).

Reply:
Done.

Reply:
You still haven't split up the tests yet. Specifically, what I mean is to break this method up so that you have multiple `def test_....`

Reply:
Done